src/main/grammars/RustParser.bnf

```diff
@@ -353,7 +353,11 @@ WherePred ::= Lifetime LifetimeParamBounds | ForLifetimes? TypeReference TypePar
 
 ForLifetimes ::= for '<' LifetimesParams '>'
 
-ExternAbi ::= extern STRING_LITERAL?
+ExternAbi ::= extern STRING_LITERAL? {
+  extends = "org.rust.lang.core.psi.ext.RsStubbedElementImpl<?>"
+  stubClass = "org.rust.lang.core.stubs.RsExternAbiStub"
+  elementTypeFactory = "org.rust.lang.core.stubs.StubImplementationsKt.factory"
+}
```

Comment:
I don't think we need to introduce a stub node for abi: we can store this info in the function stub itself, even if we have to look at the enclosing foreign mod.